docs/src/developers_guide/contributing_getting_involved.rst

```diff
@@ -22,22 +22,22 @@ Alternatively, **join the conversation** in Iris `GitHub Discussions`_, when
 you would like the opinions of the Iris community.
 
 A `pull request`_ may also be created by anyone who has become a
-**contributor** to Iris_.  Permissions to merge pull requests to the
-``main`` branch are only given to **core developers** of Iris_, this is
-to ensure a measure of control.
+|contributor|_ to Iris_.  Permissions to merge pull requests to the
+``main`` branch are only given to |core developers|_ of Iris_, this is
+to ensure a measure of control. All authors on a pull request will
+automatically be asked to sign the
+`SciTools Contributor's License Agreement (CLA)`_, if they have not already
+done so.
 
 To get started we suggest reading recent `issues`_, `GitHub Discussions`_ and
 `pull requests`_ for Iris.
 
 If you are new to using GitHub we recommend reading the
 `GitHub getting started`_
 
-.. note:: For more information on becoming a contributor_ including a link to
-          the Contributors Licence Agreement (CLA) see the
-          `Governance <https://scitools.org.uk/organisation.html#governance>`_
-          section of the `SciTools`_ ogranization web site.
-
```

Comment:
I've chosen to remove reliance on the scitools.org.uk governance docs, since our repo admin has evolved in ways that make it less relevant, and our current development culture often cuts scitools.org.uk out of the loop.

Happy to be argued against.